tools/imas/imas_wrapper.py

```diff
@@ -0,0 +1,542 @@
+# Functions to load data from IDS format and save DREAM output to IDS format
+
+import imas
+import numpy as np
+from imas import imasdef
+import logging
+import os
+import sys
+#sys.path.append (os.environ['DREAMPATH']+'/py')
+import h5py
+
+import DREAM
+from DREAM.DREAMSettings import DREAMSettings
+import DREAM.Settings.Equations.IonSpecies as Ions
+import DREAM.Settings.Equations.ElectricField as ElectricField
+import DREAM.Settings.Equations.ColdElectronTemperature as Tcold
+from scipy.interpolate import CubicSpline
+
+
+def readInIDSSlice(shot, run, tokamak, user=os.getlogin(), time=-999, log=False, setUpDream=True, wall_radius=-999):
```

Comment:
Could you add a comment describing what this function does? It is not immediately obvious how this differs from ``readInIDS()``.

Reply:
This information is given in the documentation. If you prefer, I can add it here as well.

Reply:
I think it would be nice to have the info here as well, to avoid having to jump between the code and documentation.

Reply:
Okay, I added the information to the code, for both fuctions.